packages/expo-analytics-amplitude/CHANGELOG.md

```diff
@@ -4,14 +4,18 @@
 
 ### 🛠 Breaking changes
 
+- Renamed `logEvent` method to `logEventAsync`, and `logEventWithProperties` to `logEventWithPropertiesAsync`. The previous two methods will remain included, but show warnings, so apps aren't broken when updating.
```

Comment:
Is this a breaking change then?

Reply:
yes, it would be. Although now the breaking change is going to be renaming all the methods to `<method name>Async`